### PR TITLE
test: a bunch of assorted tweaks, Saturday edition

### PR DESCRIPTION
**Pull request by mrc0mmand**
A couple of test improvements prompted by coverage reports.

### CODE REVIEW

File: src/test/test-specifier.c [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 #include "alloc-util.h"
 #include "log.h"
+#include "sd-id128.h"
```
Comment (yuwata) [inline src/test/test-specifier.c]: Please include `sd-` headers earlier.

File: src/test/test-secure-bits.c [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        assert_se(r > 0);
+        assert_se(secure_bits_to_string_alloc(r, &str) >= 0);
+        printf("%s = 0x%x = %s\n", joined, (unsigned)r, str);
+        assert_se(streq(joined, str));
+
+        str = mfree(str);
+
+        /* Empty string */
+        r = secure_bits_from_string("");
+        assert_se(r == 0);
```
Comment (yuwata) [inline src/test/test-secure-bits.c]: This is test code, hence not a big matter, but you can shorten code with `assert_se(func() >= 0);`

**Review by yuwata — APPROVED** [review]
LGTM. Several minor comments.

File: src/test/test-secure-bits.c (hunk 2 above)
Comment (mrc0mmand) [reply src/test/test-secure-bits.c]: No idea why I did it in the code below but not here :)

File: docs/ENVIRONMENT.md [hunk 3]
```diff
@@ -343,7 +343,7 @@ the journal instead of only when logging in debug mode.
 
 * `SOURCE_DATE_EPOCH` — if unset, the field of the date of last password change
```
Comment (YHNdnzj) [inline docs/ENVIRONMENT.md:344]: Add `$` here as well?

**Review by YHNdnzj — APPROVED** [review]

File: docs/ENVIRONMENT.md (hunk 3 above)
Comment (mrc0mmand) [reply docs/ENVIRONMENT.md:344]: Addressed in https://github.com/systemd/systemd/pull/27298.

**Pull request merged**